src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task09.java

```diff
@@ -34,6 +34,17 @@ public static void main(String[] args) {
     static int getPorchNumber(int numberFloors, int numberApartmentsPerFloor, int apartmentNumber) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        int entranceNumberApartments;
+        int entranceNumber;
+
+        entranceNumberApartments = numberFloors * numberApartmentsPerFloor;
+
+        entranceNumber = apartmentNumber / entranceNumberApartments;
+
+        if (apartmentNumber % entranceNumberApartments > 0){
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"